package/src/main/resources/etc/custom.properties

```diff
@@ -29,3 +29,11 @@ karaf.systemBundlesStartLevel=50
 #
 
 org.osgi.framework.bootdelegation=org.apache.karaf.jaas.boot,org.apache.karaf.jaas.boot.principal,sun.*,com.sun.*,javax.transaction,javax.transaction.*,javax.xml.crypto,javax.xml.crypto.*,org.apache.xerces.jaxp.datatype,org.apache.xerces.stax,org.apache.xerces.parsers,org.apache.xerces.jaxp,org.apache.xerces.jaxp.validation,org.apache.xerces.dom,javax.management.remote.rmi,com.yourkit.*
+
+#
+# Kafka Properties reflects Apache Camel
+#
+org.apache.operation.router.kafka.consumersCount=10
```

Comment:
The properties don't look good to me (it should be at least org.apache.unomi). Furthermore I think it's better to use a PID instead of system property.